OutOfSchool/OutOfSchool.WebApi/Models/WorkshopDTO.cs

```diff
@@ -21,10 +21,10 @@ public class WorkshopDTO : IValidatableObject
         [DataType(DataType.PhoneNumber)]
         [Required(ErrorMessage = "Phone number is required")]
         [RegularExpression(
-            @"([\d]{10})",
-            ErrorMessage = "Phone number format is incorrect. Example: 0501234567")]
-        [DisplayFormat(DataFormatString = "{0:+38 XXX-XXX-XX-XX}")]
-        [MaxLength(15)]
+            @"([\d]{9})",
+            ErrorMessage = Constants.PhoneErrorMessage)]
+        [DisplayFormat(DataFormatString = Constants.PhoneNumberFormat)]
+        [MaxLength(Constants.UnifiedPhoneLength)]
```

Comment:
I believe we have to stop adding duplicates.
Suppose we have to extract all things related to person data to the dedicated model and include it into all places instead of duplicating code.